### PR TITLE
Signed-off-by: fowlerk <keith.fowler.kf@gmail.com>

### PR DESCRIPTION
**Pull request by fowlerk**
Several minor modifications for v03.11 release:

Modified to add support for new thermostat attributes for 'audio','energy', and
'filter_subscription', even though we're not requesting these in the selection
request for the thermostat details.

Clarified output message where the latest database runtime data is equal to or greater
than the latest revision interval data returned from the Ecobee service for thermostat
details.

Also corrected bug in processing of GMail credentials file and included some missing
imports for the new mail routine for error-aborts.

### CODE REVIEW

File: media/windows/ECC-Ecobee/ECC Pycobee Data.py [hunk 1]
```diff
@@ -268,7 +287,7 @@
                                'hierarchy_set': 'HierarchySet', 'hierarchy_user': 'HierarchyUser',
                                'limit': 'LimitSetting', 'outputs': 'Output', 'page': 'Page',
                                'remote_sensors': 'RemoteSensor', 'capability': 'RemoteSensorCapability',
-#                               'reminders': 'ThermostatReminder2', 'runtime_sensor_metadata': 'RuntimeSensorMetadata',
+                               #                               'reminders': 'ThermostatReminder2', 'runtime_sensor_metadata': 'RuntimeSensorMetadata',
```
Comment (jsquyres) [inline media/windows/ECC-Ecobee/ECC Pycobee Data.py:290]: Tiny nit: don't know if you meant to do this.

**Review by jsquyres — APPROVED** [review]

**Comment by jsquyres** [comment]
Minor note: the "Signed-off-by" line is usually the last line of the commit message, not the first line.

You can get git to add the "Singed-off-by" line automatically by `git commit -s ...`.

File: media/windows/ECC-Ecobee/ECC Pycobee Data.py (hunk 1 above)
Comment (fowlerk) [reply media/windows/ECC-Ecobee/ECC Pycobee Data.py:290]: If you mean the large indent on the commented out line...sheesh, no!  Not sure how that happened but cleaned it up for the next iteration...

**Comment by fowlerk** [comment]
> Minor note: the "Signed-off-by" line is usually the last line of the commit message, not the first line.
> 
> You can get git to add the "Singed-off-by" line automatically by `git commit -s ...`.

Yeah, I used the -s switch to get the "signed-off", but I also edited the commit message to add the details of the change, and must have added them after the signed-off line.  Didn't realize there was a standard place for that..I'll try to make a mental note for the future, thx!

**Pull request merged**